src/Symfony/Component/HttpFoundation/Session/Session.php

```diff
@@ -258,4 +258,16 @@ private function getAttributeBag()
     {
         return $this->storage->getBag($this->attributeName);
     }
+
+    /**
+     * Add many attributes in session at one time
+     *
+     */
+    public function add(array $datas)
```

Comment:
I think the word "datas" is an English mistake.

Reply:
Yep, "data" is already plural.

Maybe a better choice would be a world that can take a plural form?

I don't have much context here, but maybe `$items`?

Reply:
Ditto. Just like you wouldn't pluralize "information", the same applies to "data".

Reply:
Hi everyone,
Thanks for feedback, maybe this feature will never exist. We are speaking about benefits or not..
I will change code only if Symfony core team need the feature.